src/components/checkbox/checkbox.scss

```diff
@@ -0,0 +1,163 @@
+ *    ever want.
+ * 4. Without this, checkboxes appear too small unless hard-pixel sizes are
+ *    used (ew).
+ */
+
+.c-checkbox {
+  @supports (-moz-appearance: none) or (-webkit-appearance: none) or
+    (appearance: none) {
+    /* 1 */
+    -moz-appearance: none; /* stylelint-disable-line property-no-vendor-prefix */
+    -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix */
+    appearance: none; /* 2 */
+    background-color: colors.$text-light;
+    border: sizes.$edge-medium solid currentColor;
+    border-radius: sizes.$radius_medium;
+    color: colors.$text-dark;
+    cursor: pointer;
+    flex: none; /* 3 */
+    font: inherit; /* 4 */
+    height: sizes.$toggle-medium;
+    margin: 0;
+    padding: 0;
+    position: relative;
+    transition-duration: motion.$speed-quick;
+    transition-property: background-color, border-color, color;
+    transition-timing-function: motion.$ease-out;
+    vertical-align: middle;
+    width: sizes.$toggle-medium;
+
+    /**
+     * We add the check mark via a pseudo element so we can easily apply
+     * transitions to it without requiring separate elements.
+     *
+     * 1. The icon looks kind of claustrophobic if it runs right up to the
+     *    element edge, so we re-use the border size as an inset value. This
+     *    appears much more balanced!
+     * 2. Starting state of animation.
+     */
+
+    &::after {
+      background-image: svg-load(
+        'icons/check.svg',
+        stroke=colors.$text-light-emphasis
+      );
+      background-position: center;
+      background-repeat: no-repeat;
+      background-size: contain;
+      bottom: sizes.$edge-medium; /* 1 */
+      content: '';
+      left: sizes.$edge-medium; /* 1 */
+      opacity: 0; /* 2 */
+      position: absolute;
+      right: sizes.$edge-medium; /* 1 */
+      top: sizes.$edge-medium; /* 1 */
+      transform: scale(0); /* 2 */
+      transition-duration: motion.$speed-quick;
+      transition-property: opacity, transform;
+      transition-timing-function: motion.$ease-out;
+    }
+
+    /**
+     * State: Hover
+     */
+
+    &:hover {
+      background-color: colors.$text-light-emphasis;
+      color: colors.$primary-brand;
+
+      .t-dark & {
+        color: colors.$text-dark;
+      }
+    }
+
+    /**
+     * State: Checked
+     */
+
+    &:checked {
+      background-color: currentColor;
+
+      /**
+       * End state of animation
+       */
+
+      &::after {
+        opacity: 1;
+        transform: scale(1);
+      }
+
+      .t-dark &:hover {
+        color: colors.$primary-brand-darker;
+      }
+    }
+
+    /**
+     * State: Focused
+     *
+     * We use `focus-visible` here because checkboxes do not inherently require
+     * keyboard interactions.
+     */
+
+    @include focus.focus-visible {
+      box-shadow: 0 0 0 sizes.$edge-large colors.$primary-brand-lighter;
+    }
+
+    /**
+     * State: Disabled
```

Comment:
> Originally I was going to style radio inputs as well, but I noticed we have never used those on our current site, and it increased the size of the CSS enough that I thought it seemed off

This made me wonder: do we ever use disabled checkboxes on the site?

Reply:
We do not. This might seem silly, but I like having `disabled` states in case we ever feel like enhancing a form for client-side functionality... its useful having an easy "don't click me, I'm posting data" state to switch to.

Maybe that's silly!